functions/packages.bash

```diff
@@ -138,6 +139,8 @@ exim_setup() {
     update-exim4.conf
     echo "OK"
   fi
+  zram_dependency install exim4
+  if ! cond_redirect systemctl -q daemon-reload &> /dev/null; then echo "FAILED (daemon-reload)"; return 1; fi
```

Comment:
Make sure to add this after all edits. Perhaps make it part of the function.

Reply:
I made sure, it was present in all routines I injected zram_dependency() to. Thought of making it part of course but want to avoid triggering it twice.
Not sure _why_ it was there but I guess it was for a reason....

Ok just had the idea to _replace_ all of today's daemon-reloads by zram_dependency() (with reload in there of course) ... would that work ?

Reply:
just did that so pls re-check
also added homegear

Reply:
Looks good.